### ISSUE TITLE
FRQ: Global IP count

### ISSUE THREAD

**Issue by nehalem9001** [bakape/meguca#616]
Is it possible to make a **global counter of active IPs**? I've looked into source code and so far I think that the websocket feed is activated only after the user enters a thread. But it would be nice to have number of active users on aggregate board.

**Comment by bakape** [comment]
There's `/json/ip-count` for global IP counts, but that indeed only counts people in thereads. Right now there is no need for users on board pages to connect with WebSockets, so it's needless overhead to even count them.
